.github/workflows/test_install.yml

```diff
@@ -0,0 +1,68 @@
+name: Test Installation
+
+on:
+  workflow_dispatch:
+    inputs:
+      os:
+        type: choice
+        description: 'Operating System'
+        required: true
+        options:
+          - ubuntu-latest
+          - macos-latest
+        default: 'ubuntu-latest'
+
+jobs:
+  test-install:
+    name: test installation (${{ github.event.inputs.os }})
+    runs-on: ${{ github.event.inputs.os }}
+
+    steps:
+    - name: checkout code
+      uses: actions/checkout@v2
+
+    - name: install Spice (https://install.spiceai.org)
+      run: |
+        curl https://install.spiceai.org | /bin/bash
+
+    - name: Add Spice bin to PATH
+      run: echo "/home/runner/.spice/bin" >> $GITHUB_PATH
+
+    - name: check installation
+      run: |
+        spice version
+  
+    - name: start Spice runtime
+      run: |
+        spice init app && spice run &
```

Comment:
We changed the behavior, so you'll need to cd into the `app`

Reply:
This workflow is targeting released version, so it won't work until next release

Reply:
Updated so it works for both currently released and next version (updated behavior)
```
- name: start Spice runtime
      run: |
        spice init app
        [ -d app ] && cd app
        spice run &
```